docs/datamodel/computeds.rst

```diff
@@ -0,0 +1,123 @@
+.. _ref_datamodel_computed:
+
+=========
+Computeds
```

Comment:
We have a way of having different titles in the navigation and in the actual page content. Maybe we have "Computeds" in the nav, but "Computed Properties and Links" in the actual page?